ansible/config_sonic_basedon_testbed.yml

```diff
@@ -117,6 +117,12 @@
     delegate_to: localhost
     when: local_minigraph is defined and local_minigraph|bool == true
 
+  - name: create frr.conf file in ansible minigraph folder
+    template: src=templates/frr.j2
+              dest=minigraph/frr.conf
+    delegate_to: localhost
+    when: frr_split_mode is defined and frr_split_mode|bool == true
```

Comment:
is it better to check if the split_mode is enabled in the image, if yes, then deploy the frr to the image?

Reply:
> the split_mode is enabled in the image

HI Guohan, As I know the only way to check split_mode is from DEVICE_METADATA config, which is not done by ansible right now, so we need to provide a parameter to this playbook. 

In Linkedin we have a step to push split_mode in config, but I am not sure in Azure needs it or not. Let me know if you have different check in mind.

Reply:
Marking as Resolved.